src/main/java/com/stripe/net/BaseApiRequest.java

```diff
@@ -14,6 +15,14 @@ class BaseApiRequest {
 
   @Setter private List<String> usage;
```

Comment:
Should this class be immutable? Or are we ok with it being mutable?

Reply:
Really conflicted here. I'll experiment.

Reply:
So the moral of the story is that immutability plays really badly with subclassing. I think [Immutable BaseApiRequest](https://github.com/stripe/stripe-java/pull/1721/commits/4f8883c32f302d1fe9822550b5a32629da012c9b) is an improvement, but it's a little unsatisfying because the helper to add "Usage" has to live in the child class instead of the parent. WDYT?